manifests/consumer/config.pp

```diff
@@ -0,0 +1,36 @@
+# Pulp Consumer Configuration
+class pulp::consumer::config {
+  file { '/etc/pulp/consumer/consumer.conf':
```

Comment:
What about setting owner/group/mode on these files? We have tended toward that as a best practice.